tests/monitoring/monitoring.go

```diff
@@ -378,83 +385,70 @@ var _ = Describe("[Serial][sig-monitoring]Monitoring", Serial, decorators.SigMon
 			waitUntilAlertDoesNotExist(virtHandler.downAlert)
 		})
 
-		It("VirtApiRESTErrorsBurst should be triggered when requests to virt-api are failing", func() {
-			for i := 0; i < 120; i++ {
-				cmd := clientcmd.NewVirtctlCommand("vnc", "test"+rand.String(6))
+		It("VirtApiRESTErrorsBurst and VirtApiRESTErrorsHigh should be triggered when requests to virt-api are failing", func() {
+			randVmName := rand.String(6)
+
+			triggerAlerts := func() bool {
+				cmd := clientcmd.NewVirtctlCommand("vnc", randVmName)
 				err := cmd.Execute()
 				Expect(err).To(HaveOccurred())
 
-				time.Sleep(500 * time.Millisecond)
-
-				err = checkAlert(virtApi.restErrorsBurtsAlert)
-				if err == nil {
-					return
-				}
+				err1 := checkAlert(virtApi.restErrorsBurtsAlert)
+				err2 := checkAlert(virtApi.restErrorsHighAlert)
+				return err1 == nil && err2 == nil
 			}
 
-			verifyAlertExist(virtApi.restErrorsBurtsAlert)
+			Eventually(triggerAlerts, 5*time.Minute, 500*time.Millisecond).Should(BeTrue())
```

Comment:
I would prefer to create 2 `verifyAlertExists` assertions(one for `VirtApiRESTErrorsBurst` and one for `VirtApiRESTErrorsHigh`): in case of failure we will be able to fast detect which one is failing.
Otherwise, as is, we will be informed that one of them, or both is not true.
If you want to create only one `Eventually` you can do something like:
```
Eventually(func(g Gomega) {
	cmd := clientcmd.NewVirtctlCommand("vnc", randVmName)
	err := cmd.Execute()
	g.Expect(err).To(HaveOccurred())
	g.Expect(checkAlert(virtApi.restErrorsBurtsAlert)).To(BeNil())
	g.Expect(checkAlert(virtApi.restErrorsHighAlert)).To(BeNil())
}, 5*time.Minute, 500*time.Millisecond).Should(Succeed())
```